schema/spacewalk/upgrade/susemanager-schema-4.1.4-to-susemanager-schema-4.1.5/003-rhnpackagecapability-name-index.sql

```diff
@@ -0,0 +1,2 @@
+CREATE INDEX rhn_pkg_cap_name
```

Comment:
Are you sure you want this script here?

I think you instead need it at https://github.com/uyuni-project/uyuni/tree/master/schema/spacewalk/upgrade/susemanager-schema-4.1.6-to-susemanager-schema-4.1.7